distributed/tests/test_ipython.py

```diff
@@ -8,11 +10,18 @@
 
 
 def need_functional_ipython(func):
+    save_policy = asyncio.get_event_loop_policy()
     try:
         import ipykernel  # noqa: F401
         import jupyter_client  # noqa: F401
     except ImportError:
         return pytest.mark.skip("need ipykernel and jupyter_client installed")(func)
+    current_policy = asyncio.get_event_loop_policy()
```

Comment:
I believe this should restore the AnyThread policy which ought to be the source of failures on Windows

Reply:
Didn't seem to work, possibly due to pytest triggering import while sniffing for tests. Moving to top-level fixture.